WeakAuras/Types.lua

```diff
@@ -155,6 +155,15 @@ WeakAuras.actual_unit_types_with_specific = {
   member = L["Specific Unit"]
 }
 
+WeakAuras.actual_unit_types_with_specific_and_multi = {
+  player = L["Player"],
+  target = L["Target"],
+  focus = L["Focus"],
+  pet = L["Pet"],
+  member = L["Specific Unit"],
+  multi = L["Multi-target (CLEU)"]
```

Comment:
Enough people don't know what CLEU means, that I'd rather write Combat Log in the parenthesis here. We don't use CLEU anywhere else in the codebase, or in the UI phrases.

Reply:
It's not really using the Combat Log either, so it shouldn't even have that.